cucumber/authenticators_azure/features/support/authn_azure_helper.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+# Utility methods for Azure authenticator
+
+module AuthnAzureHelper
+  include AuthenticatorHelpers
+
+  def create_and_set_azure_provider_uri_variable(value = azure_provider_uri)
+    create_and_set_azure_variable("provider-uri", value)
+  end
+
+  def retrieve_user_assigned_azure_access_token
+    retrieve_azure_access_token(retrieve_azure_token_command(is_user_assigned: true))
+  end
+
+  def retrieve_system_assigned_azure_access_token
+    retrieve_azure_access_token(retrieve_azure_token_command(is_user_assigned: false))
+  end
+
+  def retrieve_azure_token_command is_user_assigned:
```

Comment:
couldn't you just send true or false? `is_user_assigned: true/false` for readability?

Reply:
i don't understand your question. can you write down your suggestion?

Reply:
```
calling function:
retrieve_azure_access_token(retrieve_azure_token_command (false))
...

function:
def retrieve_azure_token_command is_user_assigned
```

Reply:
ah yes, it is possible. but i want it to be more explicit so the reader doesn't wonder what `false` is.